course_catalog/tasks_test.py

```diff
@@ -320,3 +335,14 @@ def test_upload_ocw_master_json(settings, mocker):
         f"9-15-biochemistry-and-pharmacology-of-synaptic-transmission-fall-2007/{course.course_id}_master.json",
     )
     assert json.loads(obj.get()["Body"].read())
+
+
+def test_process_bootcamps(mock_get_bootcamps):
+    """
+    Test that bootcamp json data is properly parsed
+    """
+    get_bootcamp_data()
+    assert Course.objects.count() == 3
+
+    get_bootcamp_data()
+    assert Course.objects.count() == 3
```

Comment:
Can you also test that some of the fields are correctly populated, including `platform`?
Speaking of `platform`, it might be good to add `Bootcamps` to `course_catalog.constants.PlatformType` though it's not necessary yet:

```
class PlatformType(Enum):
    """
    Enum for platforms
    """

    ocw = "ocw"
    mitx = "mitx"
    bootcamps = "Bootcamps"
```

Reply:
Should I create a `ResourceType` value for bootcamps as well?

Reply:
Might be more a question for @pdpinch.  I think the default "course" gets assigned to `learning_resource_type`, not sure if bootcamps should have a different type or not.

Reply:
‘Learning_resource_type’ is a field from OCW (I think there are only two possible values, “course” and “resource”). If we can’t leave it bull, then I guess we should set it to “course.”